tests/auth_tests/test_forms.py

```diff
@@ -351,6 +352,23 @@ def test_password_whitespace_not_stripped(self):
         form.is_valid()  # Not necessary to have valid credentails for the test.
         self.assertEqual(form.cleaned_data['password'], data['password'])
 
+    @override_settings(AUTH_USER_MODEL='auth_tests.CustomUserWithIntegerUsername')
+    def test_integer_field(self):
+        class CustomAuthenticationForm(AuthenticationForm):
+            class Meta(UserCreationForm.Meta):
```

Comment:
`UserCreationForm.Meta` looks like a typo to me. Perhaps you meant `AuthenticationForm.Meta`?